common/src/main/java/com/wynntils/features/wynntils/ChangelogFeature.java

```diff
@@ -24,8 +25,8 @@
 @ConfigCategory(Category.WYNNTILS)
 public class ChangelogFeature extends Feature {
     // "v0.0.2-alpha.2" is the first version with a changelog on GitHub
-    @RegisterConfig(visible = false)
-    public final Config<String> lastShownVersion = new Config<>("v0.0.2-alpha.2");
+    @RegisterConfig
+    public final HiddenConfig<String> lastShownVersion = new HiddenConfig<>("v0.0.2-alpha.2");
```

Comment:
Since we allow null configs, we should populate this with the first version the user ran. Obviously, separately.

Reply:
I'd like to take a sweep through all configs and make sure they make sense, and has a good default value.